setup-scripts/setup-tpm2-tools.sh

```diff
@@ -2,11 +2,4 @@
 
 set -euo pipefail 
 
-# libengine-tpm2-tss-openssl is found in bullseye-backports, which we add here but assign the
-# lowest possible priority to, to ensure that we only pull from it when absolutely necessary
-sudo sh -c 'echo "\ndeb http://http.us.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list'
-sudo sh -c 'echo "Package: *\nPin: release a=bullseye-backports\nPin-Priority: 1" >> /etc/apt/preferences.d/bullseye-backports'
-sudo apt update
-
-sudo apt -y install tpm2-tools qrencode
-sudo apt -y -t bullseye-backports install libengine-tpm2-tss-openssl
+sudo apt install -y tpm2-tools tpm2-openssl qrencode
```

Comment:
No more backport necessary because we're on Debian 12! Otherwise, this just shifts us from `libengine-tpm2-tss-openssl` (a package for a TPM OpenSSL _engine_) to https://github.com/tpm2-software/tpm2-openssl (a package for a TPM OpenSSL _provider_)

Reply:
@arsalansufi, to confirm, we still need to run `setup-scripts/setup-tpm2-totp.sh`, correct? (It's part of `setup-machine.sh` or `vxsuite-build-system`, depending on whether you're testing trusted build or not.)

Reply:
That's correct! We do still need to run that script